tests/mocks/phabricator/search-PHID-DIFF-test123.json

```diff
@@ -29,7 +29,11 @@
                   "commits": {
                     "commits": [
                       {
-                        "message": "Bug XXX - A second commit message"
+                        "message": "Bug XXX - A second commit message",
+                        "author": {
+                          "name": "John Doe",
+                          "raw": "John Doe <john@allizom.org>"
```

Comment:
Maybe add a test with an invalid email address?
From the code, I don't think it has an impact currently but it might in the future?!